ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewBackgroundDrawable.java

```diff
@@ -164,16 +167,32 @@ public void setBorderWidth(int position, float width) {
     }
   }
 
-  public void setBorderColor(int position, float color) {
+  public void setBorderRGB(int position, float rgb) {
```

Comment:
We should combine this and `setBorderAlpha` into a single public-facing API, e.g. `setBorderColor(int position, float rgb, float alpha)`, I don't think calling one without the other would make any sense and hints at a programming error. Feel free to keep the methods separate but make them private.